docs/testing.md

```diff
@@ -83,9 +83,10 @@ TEST_EXTERNAL_USER_BACKENDS=true \
 TEST_OCIS=true \
 OCIS_REVA_DATA_ROOT=/var/tmp/reva/ \
 BEHAT_FILTER_TAGS='~@skipOnOcis&&~@skipOnLDAP&&@TestAlsoOnExternalUserBackend&&~@local_storage'
```

Comment:
you need an extra `\ ` at the end of the line to make it possible to run the whole thing as one command
```suggestion
BEHAT_FILTER_TAGS='~@skipOnOcis&&~@skipOnLDAP&&@TestAlsoOnExternalUserBackend&&~@local_storage' \
```

Reply:
thankyou. I fixed that now